deploy/scripts/common.sh

```
@@ -0,0 +1,77 @@
+#!/bin/bash
+set -euo pipefail
+
+JAR_NAME="forgather-0.0.1-SNAPSHOT.jar"
```

Shellcheck (0.11.0)

[warning] JAR_NAME appears unused. Verify use (or export if used externally).

(SC2034)